examples/.env

```diff
@@ -0,0 +1,2 @@
+HTTPS=true
```

Comment:
This file should be added to the gitignore file. We should create an .env.example which people can copy and rename, so that we don't enforce environment variables.

Reply:
I don't think we need it all. I just wanted to keep the same port as prev examples have and HTTPS as far as I remember is strictly necessary because of workers. 

So because "people" you mentioned are our team, I think having preconfigured .env is just fine. But if you don't like it I can put this line right into `package.json`

```
"start": "crossenv HTTPS=true react-app-rewired start"
```